addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -47,7 +47,7 @@ const UrlPickerUserValueWidget = InputUserValueWidget.extend({
      * @param {OdooEvent} ev
      */
     _onWebsiteURLChosen: function (ev) {
-        $(this.inputEl).trigger('input');
+        this._value = this.inputEl.value;
```

Comment:
@fja-odoo In fact, not in 13.1/13.2 ?

Reply:
We can make it a fix in 13.1. At first I thought it was not that much of an issue but not being able to close the picker is pretty annoying. Will check if rebasing it on saas-13.1 doesn't brake anything